core/src/wallet/ripple/utils/RippleLikeUtils.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * RippleLikeUtils.cpp
+ * ledger-core
+ *
+ * Created by Pierre Pollastri on 18/05/2020.
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Ledger
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+#include "RippleLikeUtils.hpp"
+
+namespace {
+    constexpr uint64_t kRippleEpochSeconds = 946684800;
```

Comment:
k? What is that new convention? :D

Reply:
I think I am the one introducing that convention when I was working on Cosmos integration. 
It's a old convention but effective to avoid name clashing in local TU; a other way is to use upper snake case but I think this one should remain for macro

Reply:
Yep, sorry I'm working on a  C++ side project for friends on mine, and it's part of their convention I got used to it. I change it to upper snake.